### ISSUE TITLE
[Bug] Subscriptions not updating

### ISSUE THREAD

**Issue by NebulaBC** [iv-org/invidious#2399]
**Describe the bug**

After a couple of days my subscriptions stop updating, and do not start updating again until I restart my invidious server. If I go to channels I can still see the latest videos but they do not show up in my feed. I have seen this bug reported in the past, but every thread says it got fixed, but my subscriptions just will not update.

**Steps to Reproduce**
1. Set up Invidious instance on Ubuntu 20.04 VPS
2. Set up account and subscriptions
3. Wait a few days
4. Subscriptions stop updating (resubscribing to a channel does not work)

**Logs**

**Screenshots**

**Additional context**
 - Browser (if applicable): Any browser I log in on
 - OS (if applicable): Any device I log in on

**Comment by unixfox** [comment]
Duplicate of https://github.com/iv-org/invidious/issues/674

**Comment by NebulaBC** [comment]
I have read that before making this issue, and it is not a duplicate. That thread is from over two years ago, and is supposed to be fixed, I am having a separate issue.

**Comment by syeopite** [comment]
Do you have any logs that could help debug this? I'm thinking it might be due to Google returning 404 on RSS feeds which can sometimes happen, but seeing as you're able to subscribe I don't think that's the case.

**Comment by NebulaBC** [comment]
> Do you have any logs that could help debug this? I'm thinking it might be due to Google returning 404 on RSS feeds which can sometimes happen, but seeing as you're able to subscribe I don't think that's the case.

I everything works just fine, the feed just stops updating. I set up a cronjob to restart Invidious every day, but it’s been taking shorter and shorter time to stop working, to where I can not even make it a whole day anymore.
